graphql/enterprise/README.md

```diff
@@ -0,0 +1,43 @@
+# Running GraphiQL on Enterprise
+
+The GraphiQL Editor hosted on https://developer.github.com/v4/explorer/ is tied to your GitHub.com account, so in order to use this IDE with GitHub Enterprise, you'll need your own copy of GraphiQL that has access to your instance. There are a couple of options available, depending on your preference:
+
+### MacOS App
+Download the [GraphiQL App](https://github.com/skevy/graphiql-app) and you'll be able to specify the endpoint of your GitHub Enterprise instance.
+
+You can download a binary directly from the [releases](https://github.com/skevy/graphiql-app/releases) tab, but there's also a Cask for use with Homebrew which will download and install the latest release:
+`brew cask install graphiql`
+
+### Browser Client
+GraphiQL is also available as an NPM module that can be deployed to the browser. This folder includes an adaptation of the official [GraphQL NodeJS example](https://github.com/graphql/graphiql/tree/master/example) designed to be deployed to Pages on GitHub Enterprise.
+
+#### On-prem Considerations
+As GitHub Enterprise is designed to run "behind your firewall" and is sometimes deployed in environments without direct internet access, this repo is setup to host the React and GraphiQL dependencies locally.
+
+By default, this example will query against the GitHub Enterprise appliance it's hosted on. For instance, if the repo is located at `https://example.com/<username>/graphiql-pages`, the IDE will query the GraphQL API located at `https://example.com/api/graphql`. This will work whether or not subdomain isolation is enabled.
+
+
+#### Setup
+The example is already setup with all source files necessary to work with Pages on GitHub Enterprise. You can copy this folder as-is to your own instance, then [configure GitHub Pages to publish the master branch](https://help.github.com/enterprise/user/articles/configuring-a-publishing-source-for-github-pages/). A URL will be created for you automatically.
```

Comment:
@azizshamim Even with sparse-checkout it's retaining the directory structure, so you'd still have to move all the files into the root of the new repo. I adjusted the wording to be more explicit that you need the entire `graphql/enterprise` directory in its own repo.